### PR TITLE
Added command line wildcard support and dongle support

### PR DESCRIPTION
**Pull request by OsiJr**

### CODE REVIEW

File: vcConvertCMD/src/vcConvertCMD.cpp [hunk 1]
```diff
@@ -174,6 +165,8 @@ int main(int argc, const char **ppArgv)
   vdkConvertContext *pModel = nullptr;
   vcConvertCMDSettings settings = {};
 
+  settings.files.Init(256);
```
Comment (samsinsane) [inline vcConvertCMD/src/vcConvertCMD.cpp]: There's no call to `Deinit` this?

File: vcConvertCMD/src/vcConvertCMD.cpp [hunk 2]
```diff
@@ -56,19 +57,18 @@ struct vcConvertCMDSettings
...
   const char *pProxyPassword;
-  const char **ppInputFiles;
-  int32_t inputFileCount;
   int32_t srid;
   double globalOffset[3];
   bool pause;
   bool pauseOnError;
   bool autoOverwrite;
+
+  udChunkedArray<const char*> files;
```
Comment (peterAdams1014) [inline vcConvertCMD/src/vcConvertCMD.cpp]: This needs to be in a separate PR.

File: vcConvertCMD/src/vcConvertCMD.cpp (hunk 1 above)
Comment (OsiJr) [reply vcConvertCMD/src/vcConvertCMD.cpp]: Will fix.
